### PR TITLE
Z-index

### LINKED ISSUE #21: [TODO] z-index support
Need at bare minimum a way to do send to back, move to front.

### PR DESCRIPTION
**Pull request by paulomenezes**
Fixes #21

### CODE REVIEW

**Comment by paulomenezes** [comment]
It's a little bit difficult to see some difference, so I put the `fillStyle: 'solid'` when draw a rectangle, on line 454. 

Another point, is when select more than one element and press the move to front some times, these two elements swipe between then, is this expected?

**Comment by vjeux** [comment]
I was also working on the same thing. I decided to go the optimized route to do the move: https://github.com/excalidraw/excalidraw/pull/97

**Comment by paulomenezes** [comment]
Okay, is better to move instead of send to back or move to front. I'll close this.

File: src/index.tsx [hunk 1]
```diff
@@ -721,6 +739,39 @@ class App extends React.Component<{}, AppState> {
...
+
+  private sendToBack = () => {
+    const elementsWithIndexes = getSelectedElementsWithIndexes();
+
+    elementsWithIndexes.forEach(elementWithIndex => {
+      elements.splice(elementWithIndex.index, 1);
+    });
+
+    elementsWithIndexes.forEach(elementWithIndex => {
+      elements.splice(0, 0, elementWithIndex.element);
```
Comment (vjeux) [inline src/index.tsx]: This is the problem you described. If you have two elements selected [A, B], then you're going to push them in the reverse order [B, A]. So they will be "swapped"

**Pull request closed without merging**